shared/src/commonMain/kotlin/edu/usf/cutr/otp/planner/model/Itinerary.kt

```diff
@@ -0,0 +1,23 @@
+package edu.usf.cutr.otp.planner.model
```

Comment:
Can you change this package to `package edu.usf.cutr.otp.plan.model`, again to match the `/plan` API path?